improver_tests/calibration/ensemble_calibration/helper_functions.py

```diff
@@ -210,6 +210,61 @@ def setUp(self):
         ).merge_cube()
 
 
+def build_coefficients_cubelist(template, coeff_names, coeff_values):
+    """Make a cubelist of coefficients with expected metadata
+
+    Args:
+        template (iris.cube.Cube):
+            Cube containing information about the time,
+            forecast_reference_time, forecast_period, x coordinate and
+            y coordinate that will be used within the EMOS coefficient cube.
+        coeff_names (list):
+            The names of the EMOS coefficients. Each coefficient will be in a
+            separate cube within the resulting cubelist.
+        coeff_values (numpy.ndarray or list):
+            The values of the coefficients. These values will be used as the
+            cube data.
+
+    Returns:
+        coefficients (iris.cube.CubeList) - The resulting EMOS
+            coefficients cubelist.
+
+    """
+    aux_coord_and_dims = []
+
+    # add spatial and temporal coords from forecast to be calibrated
+    for coord in ["time", "forecast_period", "forecast_reference_time"]:
+        aux_coord_and_dims.append((template.coord(coord).copy(), None))
+
+    for coord in [template.coord(axis="x"), template.coord(axis="y")]:
+        bounds = [min(coord.points), max(coord.points)]
+        point = np.median(bounds)
+        new_coord = coord.copy(points=[point], bounds=[bounds])
+        aux_coord_and_dims.append((new_coord, None))
+
+    attributes = {
+        "mosg__model_configuration": "uk_det",
```

Comment:
`"mosg__model_configuration": "uk_ens"`.  If we are adding this value, let's make it sensible.

Reply:
I've updated this.